src/test/resources/com/tresys/mil-std-2045/milstd2045.tdml

```diff
@@ -1084,6 +1084,100 @@ SOFTWARE.
     </tdml:infoset>
   </tdml:parserTestCase>
   
+  <tdml:parserTestCase 
+    name="test_2045_C_minimum_size_acknoledgement" 
+    root="milstd2045_application_header"
+    model="hdr" 
+    description='Minimum size acknowledgement message.'>
+    <tdml:document bitOrder="LSBFirst">
+      <tdml:documentPart type="bits" bitOrder="LSBFirst" byteOrder="RTL"><![CDATA[Version                       0010]]></tdml:documentPart>
+      <tdml:documentPart type="bits" bitOrder="LSBFirst" byteOrder="RTL"><![CDATA[FPI Data compression type        0]]></tdml:documentPart>
+      <tdml:documentPart type="bits" bitOrder="LSBFirst" byteOrder="RTL"><![CDATA[GPI Originator address group     0]]></tdml:documentPart>
```

Comment:
I'm pretty sure that some more fields would need to be defined for this to be a real acknowledgement of anything. But this tests that the response data group works for parse/unparse, and that's the primary group of stuff called for in an acknowledgement that we have no other test for.